verilog/formatting/tree_unwrapper.cc

```diff
@@ -662,6 +662,27 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
       break;
     }
 
+    case NodeEnum::kMacroCall: {
+      if (IsTopLevelListItem(Context())) {
+        VisitNewUnwrappedLine(node);
+      } else if (DirectParentIsFlowControlConstruct(Context())) {
+        // This is a single statement directly inside a flow-control construct,
+        // and thus should be properly indented one level.
+        VisitIndentedSection(node, style_.indentation_spaces,
+                             PartitionPolicyEnum::kFitOnLineElseExpand);
+      } else {
+        // For each `uvm macro start a new unwrapped line
+        auto* ptr = dynamic_cast<verible::SyntaxTreeLeaf *>(node[0].get());
```

Comment:
Suggest a helper function in `CST` to extract the macro call identifier from the `kMacroCall` node.  This way, the `[0]` doesn't get exposed here, and it becomes self-documenting.

Reply:
Done. Added `GetMacroCallId`.